boot-qemu.sh

```diff
@@ -1,12 +1,19 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
 
 # Root of the repo
 BASE=$(cd "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")" && pwd)
 
+function green() {
+    echo -e "\033[01;32m${1}\033[0m"
```

Comment:
I've been trying to stay away from `echo -e` since it is not consider portable, especially if we are changing the hashbang to `sh`: https://github.com/koalaman/shellcheck/wiki/SC2028

I guess it isn't warning here because it is not `#!/bin/sh`.

Unless you vehemently object (and feel free to), I would change that to

```
printf '\033[01;32m%s\033[0m\n' "${1}"
```